examples/data_preprocessing/[WIP]_chexpert_dataset/Prepare_CheXpert_data.ipynb

```diff
@@ -0,0 +1,958 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Label distribution in the training set: \n",
+      "\n",
+      "Pathology                     -1.0    0.0     1.0\n",
+      "--------------------------  ------  -----  ------\n",
+      "Enlarged Cardiomediastinum   12403  21638   10798\n",
+      "Cardiomegaly                  8087  11116   27000\n",
+      "Lung Opacity                  5598   6599  105581\n",
+      "Lung Lesion                   1488   1270    9186\n",
+      "Edema                        12984  20726   52246\n",
+      "Consolidation                27742  28097   14783\n",
+      "Pneumonia                    18770   2799    6039\n",
+      "Atelectasis                  33739   1328   33376\n",
+      "Pneumothorax                  3145  56341   19448\n",
+      "Pleural Effusion             11628  35396   86187\n",
+      "Pleural Other                 2653    316    3523\n",
+      "Fracture                       642   2512    9040\n"
+     ]
+    }
+   ],
+   "source": [
+    "val_list = []\n",
+    "for cond in training_labels.columns:\n",
+    "    vals = np.array(training_labels.value_counts(subset=[cond]).sort_index(ascending = True))\n",
+    "    val_list.append([cond, vals[0], vals[1], vals[2]])\n",
+    "\n",
+    "print(\"Label distribution in the training set:\", \"\\n\")\n",
+    "print(tabulate(val_list, headers = [\"Pathology\", \"-1.0\", \"0.0\", \"1.0\"]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Label distribution in the validation set: \n",
+      "\n",
+      "Pathology                     0.0    1.0\n",
+      "--------------------------  -----  -----\n",
+      "Enlarged Cardiomediastinum    125    109\n",
+      "Cardiomegaly                  166     68\n",
+      "Lung Opacity                  108    126\n",
+      "Lung Lesion                   233      1\n",
+      "Edema                         189     45\n",
+      "Consolidation                 201     33\n",
+      "Pneumonia                     226      8\n",
+      "Atelectasis                   154     80\n",
+      "Pneumothorax                  226      8\n",
+      "Pleural Effusion              167     67\n",
+      "Pleural Other                 233      1\n",
+      "Fracture                      234      0\n"
+     ]
+    }
+   ],
+   "source": [
+    "validation_labels = validation_set.iloc[:, -13:-1]\n",
+    "\n",
+    "val_list = []\n",
+    "for cond in validation_labels.columns:\n",
+    "    vals = np.array(validation_labels.value_counts(subset=[cond]).sort_index(ascending=True))\n",
+    "    if cond != \"Fracture\":\n",
+    "        val_list.append([cond, vals[0], vals[1]])\n",
+    "    else:\n",
+    "        val_list.append([cond, vals[0], 0]) # fracture never positively appears in validation set\n",
```

Comment:
Since Fracture only has label 0 (never 1) in the validation set, the value_counts function only returns a single number instead of 2 (as is the case with the other pathologies)

The output of np.array(validation_labels.value_counts(subset=["Fracture"]).sort_index(ascending=True)) is just array([234], dtype=int64), which means that Fracture has label 0 for 234 observations in the validation set (= all of the observations in the validation set). So val[1] in this case doesn't exist.

For all other pathologies we get 2 values as output, where the first number is the number of times the pathology is labeled as 0 and the second number is the number of times the pathology is labeled as 1, e.g. for Edema: array([189,  45], dtype=int64)

I will see if I can maybe find a function that automatically returns [234, 0] for Fracture, so that I don't need the exception in the loop.